cla-backend/cla/config.py

```diff
@@ -68,7 +68,7 @@
 GITHUB_PR_NOTIFICATION = 'status+comment'
 
 # GitHub Application Service.
-GITHUB_APP_WEBHOOK_SECRET = 'webhook-secret'
+GITHUB_APP_WEBHOOK_SECRET = os.getenv("GITHUB_APP_WEBHOOK_SECRET", "")
```

Comment:
Can we load this via AWS SSM like the other params?

Reply:
this is actually how we pass from AWS SSM in my understanding(for Python side), we read them from `serversless.yaml` and then they're passed as env variables to the process.